tensorflow/python/ops/nn_impl.py

```diff
@@ -435,6 +436,63 @@ def swish(features):
   return features * math_ops.sigmoid(features)
 
 
+@tf_export("math.normalize", "linalg.normalize", "nn.normalize")
```

Comment:
Can you export just linalg.normalize?

Reply:
Sure, I just added the others to make it consistent with l2_normalize.